internal/mode/static/telemetry/collector.go

```diff
@@ -263,3 +259,55 @@ func CollectClusterID(ctx context.Context, k8sClient client.Reader) (string, err
 	}
 	return string(kubeNamespace.GetUID()), nil
 }
+
+// CollectNodeList returns a NodeList of all the Nodes in the cluster.
+func CollectNodeList(ctx context.Context, k8sClient client.Reader) (v1.NodeList, error) {
+	var nodes v1.NodeList
+	if err := k8sClient.List(ctx, &nodes); err != nil {
+		return nodes, fmt.Errorf("failed to get NodeList: %w", err)
+	}
+
+	return nodes, nil
+}
+
+type clusterInformation struct {
```

Comment:
Why not make it sharable across projects not only getting the platform but also ClusterID and Version. Aren't we doing the same thing?

Reply:
@pleshakov can I resolve this as we are not currently focused on cross platform collection sharing?

Reply:
@bjee19  👍